keyboards/umbra/hotswap/keyboard.json

```diff
@@ -0,0 +1,253 @@
+{
+    "manufacturer": "IV Works",
+    "keyboard_name": "Umbra",
+    "maintainer": "IV Works",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "A2", "pin_b": "A1", "resolution": 2}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "key_lock": true,
+        "mousekey": true,
+        "encoder": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "cols": ["B2", "A4", "A3", "A0", "F1", "F0", "C15", "C14", "C13", "B9", "B8", "B7", "A15", "B3"],
+        "rows": ["B14", "A9", "B6", "B5", "B4"]
+    },
+    "processor": "STM32F072",
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        },
+        "led_count": 17
+    },
+    "url": "https://iv-works.com/i",
+    "usb": {
+        "device_version": "0.0.5",
+        "pid": "0x0102",
+        "vid": "0x3786"
```

Comment:
This vendor ID is already in use and assigned.
https://www.the-sz.com/products/usbid/index.php?v=0x3786&p=&n=

Generally, the higher the number it is, the better.  (the 0x3### range is being actively assigned)

Reply:
Great catch! I swore I check the ID but that was a while ago. Updated!